### DISCUSSION TITLE
React project, Works but when I recharge 404

### DISCUSSION THREAD

**Discussion by federicowoodward** [community/community#69495] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I load 2 images, the first one is when i click on the gitgub pages link in my repo configuration, the second one is when i reload the page or send it to someone, it shows 404.

maybe is something from react?  

![image](https://github.com/community/community/assets/83924684/c0e1bc57-4585-4da7-bea0-0fbfdbfd5f52)
![image](https://github.com/community/community/assets/83924684/48341648-4e4e-46d2-8f30-1559565c9caf)

**Comment by prabhath-r** [comment]
Hey, I guess that is an issue related to your GitHub Pages site not being publicly accessible. Private repos can't be used for Github Pages without the subscription. So make sure your repository is public.

**Comment by vheidari** [comment]
hey @federicowoodward

Let's troubleshoot your problem.

First of all, to create a **GitHub page**, I recommend you carefully read this document on GitHub and follow its steps:
```
https://docs.github.com/en/pages/quickstart
```
If you have followed the steps in the above link and are still experiencing problems, please follow the steps below.

According to the above link, when you want to display a GitHub page under the address `federicowoodward.github.io`, you should create a new public repository under your account with the address `federicowoodward.github.io`.

I noticed that you don't have any repository on your account with the name `federicowoodward.github.io`.

If you haven't created a repository with this name on your account or accidentally deleted it, you should create it for GitHub to display your site under the URL `federicowoodward.github.io`.

However, if you have created a repository with the name `federicowoodward.github.io` and changed its visibility to private, you won't be able to see your site under the address `federicowoodward.github.io`, and you will receive a **404 error**.

To resolve this problem, you must change your repository's **visibility** from `private` to `public`.

To change the **visibility** of a repository on your account, I recommend you read this document on GitHub:
```
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/setting-repository-visibility#making-a-repository-public
```
Even after changing the **visibility** from `private` to `public`, you may still encounter a **404 error** when you check the address `federicowoodward.github.io`.

You receive a **404 error** because GitHub doesn't rebuild your site after you change its visibility from private to public.

To display your site under the address `federicowoodward.github.io`, you must rebuild your site using GitHub Actions.

But how?

There are **two** approaches that you can follow to rebuild your site using `GitHub Actions`:

### Through the Actions tab in your repository:

- Inside your repository, click on the **Actions** tab.
- Then, under the All Workflows section, find the latest **pages build and deployment** workflow with a **green** tick and click on it.
- Once you are on the address of the latest **pages build and deployment,** you should find a **Re-run all jobs** button. Press it to let GitHub Actions rebuild and deploy your site again.

### Through a new commit to your project:
- When your repository is public, for each new commit to your project, GitHub will rebuild and redeploy it. For this reason, when you are confident about your repository's visibility, you can send a new commit and push it to GitHub to rebuild your website.

**Comment by passionate2082** [comment]
Hello.
I think your project's address is not linked correctly.
